ios/FluentUI.Demo/FluentUI.Demo/Demos/LeftNavDemoController.swift

```diff
@@ -115,6 +126,7 @@ class LeftNavMenuViewController: UIViewController {
             let statusCellChild = MSFListCellState()
             statusCellChild.leadingViewSize = .small
             statusCellChild.title = presence.cellTitle()
+            statusCellChild.cellBackgroundColor = UIColor(light: .white, dark: .clear)
```

Comment:
The color inconsistency is stemmed from using `.systembackground` in `LeftNav`. I have updated this color to `system background` instead of redefining color multiple times.